main/pipeline/pipeline.cc

```diff
@@ -1061,6 +1061,14 @@ class AllNamesCollector {
         handleUnresolvedConstantLit(ctx, original.get());
         return original;
     }
+
+    unique_ptr<ast::UnresolvedIdent> postTransformUnresolvedIdent(core::Context ctx,
```

Comment:
To make 'find all refs' fast, we store a list of all Names encountered in each file, and use that as a filter for "give me all references to symbol x".

This change stores the names of ivars and gvars into that list.